src/main/java/appeng/util/Platform.java

```diff
@@ -1887,15 +1887,51 @@ public static String formatByteDouble(final double n) {
         return (n / BYTE_LIMIT[0]) + " " + BYTE_UNIT[0];
     }
 
-    public static String formatNumberLong(final long n) {
-        if (n > 1_000) {
-            for (int i = 1; i < NUM_LIMIT.length; i++) {
-                if (n < NUM_LIMIT[i]) {
-                    return String.valueOf(n / NUM_LIMIT[i - 1]) + " " + NUM_UNIT[i - 1];
-                }
-            }
+    /**
+     * From large double to num with unit
+     * 
+     * @param n number wait to format
+     * @return String
+     */
+    public static String formatNumberLong(final double n) {
```

Comment:
This change *breaks* existing method: original method is designed for `long` argument, but you change it to `double` (but keeping old name!) Please, revert all these changes and restore original method.

If you need method for formatting double values then create *new* one. And, please use new `DecimalFormat` instead of inventing bicycle.